hikari/api/special_endpoints.py

```diff
@@ -2086,6 +2207,10 @@ def add_select_menu(
         `MessageActionRowBuilder.add_channel_menu` and
         `MessageActionRowBuilder.add_text_menu`.
 
+        .. deprecated:: 2.0.0.dev123
```

Comment:
Just leaving this here as a note that these versions have to be bumped before this is merged